module/VuFind/src/VuFind/RecordDriver/SolrMarc.php

```diff
@@ -72,4 +75,69 @@ public function getURLs()
             $this->getIlsURLs()
         );
     }
+
+    /**
+     * Takes a Marc field that notes are stored in (ex: 950) and a list of
+     * sub fields (ex: ['a','b']) optionally as well as what indicator
+     * numbers and values to filter for and concatenates the subfields
+     * together and returns the fields back as an array
+     * (ex: ['subA subB subC', 'field2SubA field2SubB'])
+     *
+     * @param string $field    Marc field to search within
+     * @param array  $subfield Sub-fields to return or empty for all
+     * @param array  $indData  Array containing the indicator number as the key
+     *                         and the value as an array of strings for the
+     *                         allowed indicator values
+     *                         ex: [['1' => '1', '2', '2' => '']]
```

Comment:
I think you're missing some brackets in this example:

```suggestion
     *                         ex: [['1' => ['1', '2'], '2' => ['']]]
```